src/account/accountReducers.js

```diff
@@ -79,9 +79,8 @@ export default (state = initialState, action) => {
       ];
     }
     case LOGOUT: {
-      const newState = state.slice();
-      newState[0].apiKey = '';
-      return newState;
+      // Remove the active account
```

Comment:
I can leave it in. We should still land users to the 'realm' screen though because otherwise we won't have the auth backends available. (Alternatively we'd need to store the auth backends in `state` but then they might be stale, etc..)